microprofile.cpp

```diff
@@ -147,6 +148,20 @@ void* MicroProfileReallocInternal(void* pPtr, size_t nSize);
 void* MicroProfileAllocAligned(size_t nSize, size_t nAlign);
 void MicroProfileFreeAligned(void* pMem);
 
+#ifdef MICROPROFILE_PS4
+#define MICROPROFILE_PS4_DECL
+#include "microprofile_ps4.h"
+#endif
+
+#ifdef MICROPROFILE_XBOXONE
+#define MICROPROFILE_XBOXONE_DECL
+#include "microprofile_xboxone.h"
+#else
```

Comment:
You cannot move these to the .cpp,  because they contain definitions needed in the header.
Please leave the _PS4/_XBOXONE part of the code, as it is does not have a build test

Reply:
otherwise, everything looks fine

Reply:
I didn't move them to the cpp - these are the MICROPROFILE_*_DECL lines, I moved them from lines 299-309 up to lines 151-164. I can't recall why I did this (this PR is for changes I did a long while ago) but removing this change doesn't seem to affect compilation now so I'll move them back.